pulsar-broker/src/main/java/com/yahoo/pulsar/broker/admin/PersistentTopics.java

```diff
@@ -257,6 +257,23 @@
         }
     }
 
+    protected void validateAdminAndClientPermission(DestinationName destination) {
+        try {
+            validateAdminAccessOnProperty(destination.getProperty());
+        } catch (Exception ve) {
```

Comment:
Mino thing: do you think it would be easy to not rely on exceptions on the normal cases? Exception can be expensive to throw (basically it needs to generate the whole stack trace). 

If that requires lot of refactoring, let's go ahead with this change.